README.md

```diff
@@ -2,12 +2,36 @@ Build Status: [![Build Status](https://azure-functions.visualstudio.com/azure-fu
 
 CoreTools Healthcheck: [![CoreTools healthcheck](https://azure-functions.visualstudio.com/azure-functions-docker/_apis/build/status/v3%20core-tools%20image%20health%20check?branchName=dev)](https://azure-functions.visualstudio.com/azure-functions-docker/_build/latest?definitionId=43&branchName=dev)
 
+# Host Release Information
+
+Azure-Functions-Docker/host Images : 
+
+Azure-Functions-Docker releases Linux Container Images for Azure-Functions-Host.  These images consist of the[ Azure-Functions-Host](https://github.com/Azure/azure-functions-host), [Extension Bundles](https://github.com/Azure/azure-functions-extension-bundles), and a language worker (ex. [Powershell-Worker](https://github.com/Azure/azure-functions-powershell-worker)).  
```

Comment:
I think functions host automatically brings in some other language workers as well 
Ex: https://github.com/Azure/azure-functions-host/commit/9cf4d03e54bcc98cc5b5e165467b56d04d0b6f99#diff-c0ba1fae310774f3dc1bf6b5919e9f9b9bafb76ec834ae9c3a372a196ce62a3f
(dont have the complete list)

Reply:
That is correct. All the officially supported languages - language workers as of now ship with Host